ci/scripts/bazel-coverage.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+bazel query --universe_scope=//... \
```

Comment:
I'll look whether there's any change with having functional rust code coverage with wasm32 target in the follow up.